ci-operator/step-registry/ovn/sdn-migration/feature-migration/config-multicast/ovn-sdn-migration-feature-migration-config-multicast-commands.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -x
+set -o errexit
+set -o nounset
+set -o pipefail
+
+
+# I'm unsure if netnamespace will already exist, maybe i only need to patch it.
+oc create ns test-migration
```

Comment:
wanna add your "|| true" here as well? and remove the "maybe" part of the comment. just note that you
aren't sure if it's there or not so just doing it anyway, since it can't hurt

Reply:
Leaving this unresolved for now so I don't forget to remove that "maybe" part of the comment later...